openapi/data_repository_service.swagger.yaml

```diff
@@ -160,36 +159,23 @@ securityDefinitions:
     name: Authorization
     in: header
 definitions:
-  SystemMetadata:
```

Comment:
SystemMetadata and UserMetadata are still being referenced in the Bundle definition, so the documentation build is failing

Reply:
References are removed in PR #244, lines [244](https://github.com/ga4gh/data-repository-service-schemas/pull/244/files#diff-4bd72006051046bcde21e06812a2c330L244) and [246](https://github.com/ga4gh/data-repository-service-schemas/pull/244/files#diff-4bd72006051046bcde21e06812a2c330L246)

Reply:
Got it -- unfortunately splitting it across PRs means that this one fails the Travis checks. Do you plan to merge them somehow, or would you like me to review some other PRs first, so those can be merged and then we can come back to this one?

Reply:
I'll remove the references here so that the build passes and this could be merged without conflict. Once merged PR #244 should go through as it makes the same deletions.